### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA announces additional inappropriate information when "Browse Repositories" dialog is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1484]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to and select File->Settings.
3. In "settings" dialog , tab to and select plugin->browse repositories button.
4. "Browse Repositories" dialog will open.
5. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Browse Repositories" dialog is opened 

**Jaws  behaviour:**
 Browse Repositories dialog
Sort by: name VCS INTEGRATION .ignore7174454 downloads Updated 11-01-2018 v2.4.0 
table

**NVDA Behaviour:**
Browse Repositories  dialog  Sort by: name
VCS INTEGRATION

.ignore
7174454 downloads
Updated 1/11/2018
v2.4.0
table
selected  row 1  column 1

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e Browse Repositories dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![browse repositories-opened](https://user-images.githubusercontent.com/22956130/36106086-58d24658-103c-11e8-970f-fcbb489bb69e.jpg)
![browse repositories-nvda](https://user-images.githubusercontent.com/22956130/36106092-5cf9b9be-103c-11e8-9640-aa7ca0227679.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749960
